tests/rpm-ostree/initramfs.yml

```diff
@@ -53,9 +53,13 @@
   set_fact:
     osname: "{{ ros_booted['osname'] }}"
 
-- name: Get bootloader entry
+- name: Get boot entry file
+  shell: ls -1 /boot/loader/entries/ | grep -E "ostree-[2-]*fedora-atomic[-0]*.conf"
```

Comment:
Is this going to work for non Fedora streams?

Reply:
:man_facepalming: I was testing with fedora